easybuild/easyconfigs/i/icc/icc-2016.0.109-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+
+name = 'icc'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_2016_composer_edition_for_cpp_2016.0.109.tgz']
```

Comment:
As I said, I'm not sure what the best long term approach is here, but for now, I'd stick to the 'official' tarball name (without the `_2016.0.109` part), but add the checksum in the easyconfig:

```
checksums = ['f57a892fb494db3c80f20a88aa3e901f']
```

If we notice that Intel isn't properly versioning the tarballs with a future release, we can change this easyconfig accordingly and require renaming the downloaded tarball.